server/models/Notification.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const notificationSchema = new Schema({
+    
+    receiver: { type: mongoose.Schema.Types.ObjectId, ref: 'User' },
+    notifications: [{
+        type: {
+            type: String,
+            enum: ['serviceRequest', 'serviceAvailable'], 
+        },
+        sender: {type:mongoose.Schema.Types.ObjectId, ref:'User'}, 
+        title: {type: String},
+        message: {type: String},
+        isRead: {type: Boolean,default: false},
+        created_at:{type: Date, default: Date.now},
+    }
+        
+  ]
```

Comment:
I think this code needs to be properly formatted.

Reply:
Sure 👍